packages/teams-js/src/public/dialog.ts

```diff
@@ -125,7 +125,7 @@ export namespace dialog {
    * @beta
    */
   export function submit(result?: string | object, appIds?: string | string[]): void {
-    ensureInitialized(FrameContexts.content, FrameContexts.sidePanel, FrameContexts.task, FrameContexts.meetingStage);
+    ensureInitialized(FrameContexts.content, FrameContexts.task);
```

Comment:
>FrameContexts.content [](http://example.com/codeflow?start=22&length=21)

You should probably add a comment here that says something like "This should not be here, but it's only here because Teams mobile incorrectly returns FrameContext.content. When that bug is fixed, this will be removed."

Reply:
This might be overkill but you could probably make it so that `FrameContexts.content` is only allowed when running on Teams Android/iOS (or wherever the bug exists) and that it's not allowed elsewhere.

Reply:
Adding a comment is a good idea! I can look into restricting context to ios/mobile after UX. If it takes so much time, and feels like an over kill, we can leave it